sync/src/lib.rs

```diff
@@ -50,7 +50,7 @@
 //! 	);
 //! 	let client = Client::new(
 //!			ClientConfig::default(),
-//!			ethereum::new_frontier(),
+//!			&ethereum::new_frontier(),
```

Comment:
it wasn't _directly_ related, although it was useful to assist in adding the `SnapshotService` to `ClientService` without loading the spec twice. i will refactor it out into another PR.